src/ImageSharp/Formats/Gif/GifDecoderCore.cs

```diff
@@ -332,7 +344,7 @@ private void ReadApplicationExtension(BufferedReadStream stream)
             if (subBlockSize == GifConstants.NetscapeLoopingSubBlockSize)
             {
                 stream.Read(this.buffer.Span, 0, GifConstants.NetscapeLoopingSubBlockSize);
-                this.gifMetadata!.RepeatCount = GifNetscapeLoopingApplicationExtension.Parse(this.buffer.Span.Slice(1)).RepeatCount;
+                this.gifMetadata!.RepeatCount = GifNetscapeLoopingApplicationExtension.Parse(this.buffer.Span[1..]).RepeatCount;
```

Comment:
```suggestion
                this.gifMetadata!.RepeatCount = GifNetscapeLoopingApplicationExtension.Parse(this.buffer.Span.Slice(1)).RepeatCount;
```
Perf-wise `Slice` is [better](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunHiUGAMQgQAFAGUADtgB2AHmABPDDAB8DXNJkBKLj07UeRjVoYBeE7IB04gDYBLMDFHltAbj3cAvp4a/6TAIAQthQEloKymqWOr4GxjyasuYxANrkVlYAuh6GPD7UXkA==) than the range-operator. Unfortunately Roslyn compiles "bad" IL here, which the JIT doesn't optimize away.

So either
* keep Slice
* wait for Roslyn to be fixed (which takes some years already for that issue)

Reply:
I'm just taking the hit here as we have elsewhere. It's annoying but worth the readability improvements.

Reply:
I don't think this is a critical path, simpler looking code is better.